lighthouse/lighthouserc.js

```diff
@@ -59,6 +61,23 @@ const AUDIT_URLS = {
       'https://www.test.bbc.com/pashto/afghanistan-52643309?renderer_env=live',
     ],
   },
+
+  TOPIC_PAGE: {
+    live: ['https://www.bbc.com/pidgin/topics/c0823e52dd0t'],
+    test: ['https://www.test.bbc.com/pidgin/topics/c0823e52dd0t'],
+  },
+
+  HOME_PAGE: {
+    live: ['https://www.test.bbc.com/kyrgyz?renderer_env=live'],
+    test: ['https://www.test.bbc.com/kyrgyz'],
+  },
+
+  MEDIA_ARTICLE_PAGE: {
+    live: ['https://www.bbc.com/hausa/articles/cw5kgkr4wvzo'],
+    test: [
+      'https://www.test.bbc.com/hausa/articles/cw5kgkr4wvzo?renderer_env=live',
```

Comment:
Using `renderer_env` to display live data (instead of using test data which can be very out of date)